mosaic-runtime/src/commonMain/kotlin/com/jakewharton/mosaic/canvas.kt

```diff
@@ -32,41 +33,61 @@ internal class TextSurface(
 
 	override operator fun get(row: Int, column: Int) = rows[translationY + row][translationX + column]
 
-	fun appendRowTo(appendable: Appendable, row: Int) {
+	fun appendRowTo(appendable: Appendable, row: Int, ansiLevel: AnsiLevel) {
```

Comment:
Can we accept this parameter in the constructor of the surface? It feels weird to be able to alter the ANSI level per row.